frontend/html/posts/common/upvote.html

```diff
@@ -0,0 +1,5 @@
+{% if me != post.author %}
```

Comment:
```suggestion
{% if me and me != post.author %}
```
Ох, только увидел. Тут не хватает проверки на то, что me это не None. Иначе незалогиненные юзеры тоже смогут голосовать